tests/test_server_basic.py

```diff
@@ -1420,6 +1428,7 @@ def test_default_values(dispatcher_live_fixture, additional_parameter):
 
     if additional_parameter:
         params['additional_param'] = 'no_value'
+        params['allow_unknown_args'] = True
```

Comment:
do we have test when it's not allowed?

Reply:
In this specific test, it doesn't make sense, but it's not allowed by default. 
In test_unknown_argument the behavior is tested directly (but need to uncomment). There are also changes for several tests where job_id is asserted

Reply:
yeah I meant elsewhere. Uncomment is good.